stock_orderpoint_automatic_creation/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Stock Orderpoint Automatic Creation
+===================================
+
+This module generates stock orderpoints automatically for a product when is
+created taking created user company as reference.
```

Comment:
s/taking created user company/taking the company of the user who created the product/